client/components/StudentManagement.react.js

```diff
@@ -36,6 +36,17 @@ class StudentManagement extends React.Component {
     return res.data.success;
   };
 
+  handleDelete = async studentIdToDelete => {
+    const res = await axios.delete(`api/students/delete/${studentIdToDelete}`, {
+      studentId: studentIdToDelete
+    });
+    if(res.data.success) {
+      this.getStudents();
```

Comment:
we should avoid HTTP requests wherever possible - in this case we can just remove it from our state